euclidean_gcd/EuclideanGCD.java

```diff
@@ -16,18 +16,16 @@ static int euclideanGCD(int first, int second) {
     }
 
     static int euclideanGCDRecursive(int first, int second) {
-        if(second == 0)
-            return first;       // First becomes gcd if second becomes zero
-        else
-            return euclideanGCDRecursive(second, (first % second));
+        return (second == 0) ? first : euclideanGCDRecursive(second, (first % second))
```

Comment:
You missed the semicolon. Please make sure that you run the code before commiting.